drivers/scheduler/scheduler.go

```diff
@@ -347,6 +347,9 @@ type Driver interface {
 	// SetConfig sets connnection config (e.g. kubeconfig in case of k8s) for scheduler driver
 	SetConfig(configPath string) error
 
+	// SetGkeConfig sets connnection config (e.g. kubeconfig in case of GKE cluster) for scheduler driver
+	SetGkeConfig(configPath string, gkeCredString string) error
```

Comment:
We can call this `jsonKey` instead of `gkeCredString` because it is called the same in Px Backup